docs/proposal-platforms.md

```diff
@@ -277,10 +278,33 @@ suite-run functionality:
          using a wide range of sub-directories such as `python`, `util`, `data`
          (although it is not clear how many of these would need to be installed
          on remote platforms).
-       * Proposal: we allow the include list to be configurable at the global
-         level (to cater for old suites which need this) but encourage users to
-         adapt their workflows to use the supported directories and, therefore,
-         don't allow this to be configured at the suite level.
+       * If we allow global configuration this is likely to lead to portability
+         issues where a suite only works if you modify your global config. If
+         additional items need to be installed it's better to force these to be
+         configured in the suite. Note that this will require changes to some
+         Cylc 7 suites in order for them to work at Cylc 8.
+       * Hopefully it should be fine to install the same files to all install
+         targets. If there are cases where this is an issue we can recommend the
+         use of install tasks as a workaround.
+       * Proposal:
+         * Install the same items to all install targets.
+         * Allow the installation items to be configured at the suite level via
+           ones of the following ways:
+           * A new setting `[scheduler]install` with a default value of
+             `app bin etc lib` (the items in `.service` are always installed).
+             Note that if we want to be able to support copying top level files
+             as well as directories then we would need to do something like
+             `app/ bin/ etc/ lib/ extra-dir-to-copy/ extra-file-to-copy`.
```

Comment:
Note that `[scheduler]install` should not accept patterns or anything other than top-level files or directories.

Reply:
Why is that exactly?

Reply:
1) Force adoption of standard permitted directories (`bin`, `lib`, `app`).
2) Prevent overly generic globs (e.g. `*`).
3) Multi-level includes would require extra logic in forming the `rsync` command.

Reply:
I can't see any reason not to allow patterns so long as we explicitly exclude anything which should never be copied. This makes it easy for users to mimic the current `rose suite-run behaviour`. See updated proposal.

Reply:
Resolving this - please comment on the updated proposal